source/apiVolontaria/setup.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+import os
+from setuptools import find_packages, setup
+
+with open(os.path.join(os.path.dirname(__file__), 'README.rst')) as readme:
+    README = readme.read()
+
+# allow setup.py to be run from any path
+os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
+
+setup(
+    name='django-api-volontaria',
+    version='1.0',
+    packages=find_packages(),
+    include_package_data=True,
+    license='BSD License',
+    description='API to manage volunteer and order in an organisation.',
+    long_description=README,
+    url='https://volontaria.github.io/',
+    author='Noel Rignon',
+    author_email='rignon.noel@gmail.com',
+    classifiers=[
+        'Environment :: Web Environment',
+        'Framework :: Django',
+        'Framework :: Django :: X.Y',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: BSD License',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Topic :: Internet :: WWW/HTTP',
+        'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
+    ],
+    install_requires=[
+        'Django==1.11.10',
+        'django-filter==1.1.0',
+        'djangorestframework==3.7.1',
+        'Markdown==2.6.11',
+        'drfdocs==0.0.11',
+        'django-cors-headers==2.1.0',
+        'Faker==0.8.12',
+        'factory-boy==2.10.0',
+        'imailing==1.0.0',
+        'django-import-export==1.0.0',
+    ],
```

Comment:
We need to find a way to integrate the requirements from a unique source and not copy paste them here and in the `requirements.txt`